homework-midy/dogPhotoGallery/index.js

```diff
@@ -0,0 +1,40 @@
+const url = 'https://dog.ceo/api/breeds/image/random';
+// ------------------------XMLHttpRequest---------------------
+function xmlRequest() {
+  const xhr = new XMLHttpRequest();
+  xhr.onreadystatechange = () => {
+    if (xhr.readyState === XMLHttpRequest.DONE) {
+      const imageListItem = document.createElement('li');
+      const image = document.createElement('img');
+      image.setAttribute('src', JSON.parse(xhr.response).message);
+      imageListItem.appendChild(image);
+      document.getElementById('dog-list').appendChild(imageListItem);
+    }
+  };
+  xhr.open('GET', url);
+  xhr.send();
+}
+
+document.getElementById('xml').addEventListener('click', () => {
+  xmlRequest();
+});
```

Comment:
No need to call a function with no arguments that calls a function with no arguments
```suggestion
document.getElementById('xml').addEventListener('click', xmlRequest);
```